tailwind.config.js

```diff
@@ -6,7 +8,18 @@ export default {
     "./.storybook/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        ...defaultColors,
+        'orange':{
+          DEFAULT: '#ff6725',
+        'dark': '#ff3f25',
+        'light': '#fef7f3',
+        },
+        'brown': '#453a34',
+        'green': '#00b579',
```

Comment:
Indentation is incorrect. We should always match the brackets and lines correct. This is a common issues across multiple PRs so if you need more help in understanding this please message me.

Reply:
I think I have done this one too